lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidator.test.js

```diff
@@ -0,0 +1,416 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../../tests/utils/run-serverless');
+const fixtures = require('../../../../../../../../tests/fixtures');
+
+const expect = chai.expect;
+chai.use(require('chai-as-promised'));
+
+describe('#compileRequestValidators()', () => {
+  let serverlessInstance;
+
+  before(() => {
+    return Promise.resolve(
+      runServerless({
+        cwd: fixtures.map.requestSchema,
+        cliArgs: ['package'],
+      })
+    ).then(serverless => (serverlessInstance = serverless));
+  });
+
+  after(() => {
+    fixtures.cleanup();
+  });
+
+  describe(' reusable schemas ', () => {
+    it('Should process schema from apiGateway provider, full config', () => {
+      const cfResources =
+        serverlessInstance.serverless.service.provider.compiledCloudFormationTemplate.Resources;
+      const naming = serverlessInstance.serverless.getProvider('aws').naming;
```

Comment:
It'll be nice to get both of this directly from `runServerless` result. See: https://github.com/serverless/test/blob/master/docs/run-serverless.md#result-values (it's also used in other tests across the project)